packages/embark-contracts-manager/src/index.js

```diff
@@ -535,12 +535,12 @@ class ContractsManager {
       }
     ], function (err) {
       if (err) {
-        self.compileError = true;
+        self.compileError = true; // TODO: WUT
```

Comment:
I think we should set this as the error itself. It would make more sense than a boolean. Especially since `false` makes async waterfall fucks up with `false`